acceptance/tests/environment/environment_scenario-bad.rb

```diff
@@ -16,32 +16,32 @@
 apply_manifest_on(master, environment_manifest(testdir), :catch_failures => true)
```

Comment:
i added agent roles to the master in our testsuite and with these changes the tests run and pass.  It fixes the issue where the non-master commands were not running if master != agent but also should fix the issue you are seeing.
it runs and passes against whatever server is installed in our aio acceptance suite.